VideoLocker/build.gradle

```diff
@@ -170,8 +170,8 @@ android {
         // minimum version is Android 4.0
         minSdkVersion 14
         targetSdkVersion 21
-        versionCode 74
-        versionName "2.1.1"
+        versionCode 77
```

Comment:
I just saw that there was a sudden jump of the version code from 74 to 77 :frowning: